waku/v2/protocol/filter/filter_push_test.go

```diff
@@ -16,7 +16,7 @@ func (s *FilterTestSuite) TestValidPayloadsASCII() {
 	s.subDetails = s.subscribe(s.testTopic, s.testContentTopic, s.fullNodeHost.ID())
 
 	// Prepare data
-	messages := prepareData(100, false, false, true, tests.GenerateRandomASCIIString)
+	messages := prepareData(50, false, false, true, tests.GenerateRandomASCIIString)
```

Comment:
> I've reverted related commit. I'll create dedicated PR later. It would need to be merged before this PR. Not sure why Github runners become slower. [d6fb227](https://github.com/waku-org/go-waku/commit/d6fb227372a299eeec1c27655b10581fbdf53934)

Ah, ok...in that case it is fine.